middleman_protocol/tests/test_stream_async.py

```diff
@@ -0,0 +1,95 @@
+from asyncio import StreamReader
+from asyncio import StreamWriter
+
+import pytest
+from assertpy import assert_that
+from mock import MagicMock
+from mock import Mock
+
+from middleman_protocol import constants
+from middleman_protocol import exceptions
+from middleman_protocol.constants import ESCAPE_CHARACTER
+from middleman_protocol.constants import ESCAPE_SEQUENCES
+from middleman_protocol.constants import FRAME_SEPARATOR
+from middleman_protocol.stream_async import handle_frame_receive_async, send_over_stream_async
+from middleman_protocol.stream_async import map_exception_to_error_code
+
+SOME_BYTES = b'1234567890qwerty'
```

Comment:
Please use function `os.urandom(bytes_length)`

Reply:
Won't/can't do. I've heard that it's bad practice to use random values in tests, if You can hard code them.

Reply:
@Jakub89 is right. You can use random numbers but if you do, you should always run with the same seed so that they're reproducible in separate test runs. Otherwise you may get non-deterministic tests.